src/routers/alpha-router/alpha-router.ts

```diff
@@ -1471,7 +1471,60 @@ export class AlphaRouter
     const V3poolsRaw = V3poolAccessor.getAllPools();
     const V2poolsRaw = V2poolAccessor.getAllPools();
 
-    const poolsRaw = [...V3poolsRaw, ...V2poolsRaw];
+    /// not just [Protocol.MIXED]
```

Comment:
What is this comment? This only gets called in `getMixedRouteQuotes` so it should only be when we do specify we want mixed routes

Reply:
I think it meant that we should apply the heuristic below when the config is not just `Protocol.MIXED`